tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/OnBehalfOfServicePrincipalTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.Identity.Test.Integration.Infrastructure;
+using Microsoft.Identity.Test.Integration.net45.Infrastructure;
+using Microsoft.Identity.Test.Integration.NetFx.Infrastructure;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.HeadlessTests
+{
+    [TestClass]
+    public class OnBehalfOfServicePrincipalTests
+    {
+        //The following client ids are for applications that are within PPE
+        private const string OBOClientPpeClientID = "9793041b-9078-4942-b1d2-babdc472cc0c";
```

Comment:
can we move to use lab's ppe tenant?

Reply:
I guess yes; I assume it'd just have to be enabled / set up for service principal OBO. @trwalke set up this one.